cornucopia/src/parser.rs

```diff
@@ -143,6 +143,11 @@ impl Query {
     fn sql_escaping() -> impl Parser<char, (), Error = Simple<char>> {
         // https://www.postgresql.org/docs/current/sql-syntax-lexical.html
 
+        // ::bind
+        let cast = just("::")
+            .then(ident())
+            .then(just("[]").repeated().at_most(1))
+            .ignored();
```

Comment:
We can do something simpler by simply ignoring `::`, the other characters being ignored anyway
```rs
let cast = just("::").ignored();
```

Reply:
Makes sense, I'll make the change.